### DISCUSSION TITLE
Visual Editor demo

### DISCUSSION THREAD

**Discussion by wwwillchen** [google/mesop#56] [Show and tell]
https://github.com/google/mesop/assets/7344640/c5ba1b61-3ba0-4733-ada3-df7e6b81246b

**Comment by simkimsia** [comment]
I saw PR #31 and it was marked as closed. Does this mean this is live? I am a bit confused as to how to use it

**Reply by wwwillchen** [reply]
We removed support for visual editor so you can't use it right now. We're thinking about re-launching it in the future. Thanks.
